### ISSUE TITLE
I get cors issue whenever I try to use private channel

### ISSUE THREAD

**Issue by Mrbeyond** [beyondcode/laravel-websockets#431]
Access to XMLHttpRequest at 'http://127.0.0.1:8000/laravel-websockets/auth' from origin 'http://127.0.0.1:3000' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.

I get the error whenever I listen to private channel where as the public channel works fine.

**Comment by rennokki** [comment]
Duplicate of https://github.com/beyondcode/laravel-websockets/issues/403#issuecomment-654762542
